charts/incubator/miniflux/Chart.yaml

```diff
@@ -21,24 +21,17 @@ keywords:
 - news
 dependencies:
 - name: common
-  version: 11.1.2
+  version: 14.3.5
```

Comment:
```suggestion
  version: 15.3.1
```

Reply:
updated in 3a89517d49908b4c067a466f4114c6e6a1eaa41b.